src/components/mainPage/CatalogSidebar.jsx

```diff
@@ -15,6 +15,7 @@ const CatalogSidebar = () => {
 
   const onClearFiltyers = () => {
```

Comment:
Typo in word "filters". Please check project wide

Reply:
Agree. Already fixed. Thanks.